src/templates/template.h

```diff
@@ -141,6 +141,7 @@ Q_OBJECT
 	 */
 	enum ScribbleOption
 	{
+		NoScribbleOptions = 0,
```

Comment:
Yes, I had this line in the code but then I removed it. The "no options" is equivalent to newly initialized `QFlags` or just plain `{}`. That is why I removed the "no bits set" before submission.

Your proposal made me look at `QFlags` usage in Qt (`fgrep -B15 -r Q_DECLARE_FLAGS /usr/include/qt5/`). I can see that zero appears in the enums in about half of the cases. That said, I don't mind re-introduction of the value. This comment is a side note explaining why it was "missing".

Reply:
Yes, this is is just a name for `ScribbleOptions{}`. However, a single-valued enum feels strange to mee, at least outside of C++ meta programming.